app/views/govuk_publishing_components/components/docs/step_by_step_nav.yml

```diff
@@ -72,6 +72,40 @@ examples:
           ]
         }
       ]
+  with_google_analytics_4_tracking:
+    description: |
+      The ga4_tracking boolean allows you to add Google Analytics 4 (GA4) tracking to your component.
```

Comment:
Are these lines meant to be separate? You need to put a blank line between them for them to render as separate paragraphs in the component guide.

Reply:
Nope, it was just to make it a bit easier to read in the code